task/maven/0.2/maven.yaml

```diff
@@ -106,6 +118,17 @@ spec:
           sed -i "s|<!-- ### HTTP proxy from ENV ### -->|$xml|" $(workspaces.maven-settings.path)/settings.xml
         fi
 
+        if [ -n "$(params.SERVER_USER)" -a -n "$(params.SERVER_PASSWORD)" ]; then
+          xml="<server>\
```

Comment:
neat, i think you can use a EOF type string

Reply:
Could you provide a detailed explanation or an example?

Reply:
https://stackoverflow.com/questions/1167746/how-to-assign-a-heredoc-value-to-a-variable-in-bash